frontend/src/lib/internalMetrics.ts

```diff
@@ -25,12 +31,13 @@ export async function captureInternalMetric(payload: InternalMetricsPayload): Pr
     }
 }
 
-export async function captureTimeToSeeData(teamId: number, payload: TimeToSeeDataPayload): Promise<void> {
+export async function captureTimeToSeeData(teamId: number | null, payload: TimeToSeeDataPayload): Promise<void> {
```

Comment:
should we return early from the method if the `teamId` is null?

Reply:
In practice this cannot be null, as prior queries would have failed - it's a smell from logic typing :(